schema/srem_schema_1-0-0.json

```diff
@@ -46,10 +46,31 @@
     "message": {
       "type": "object",
       "required": [
+        "protocol_version",
+        "station_id",
```

Comment:
What's the PDU header?

What's the difference between those fields, and the top-level `version` and `source_uuid` ones?

Why does this need to be a separate commit and can't be included in the first one?

Reply:
This is common to all ETSI ITS messages, and we have always included it at the top level of the `message` object.
`version` and `source_uuid` are Orange-specific and found in our top-level header, while `protocol_version` and `station_id` are ETSI-specific and found in the `message` section (could be needed if someone needs to translate our JSON messages into ASN1 UPER).

As for the separate commit, it is indeed not needed, I'll fix it.